### DISCUSSION TITLE
Not seeing Lanes in beta 2024-06-13-beta.1 [Transportation]

### DISCUSSION THREAD

**Discussion by BruceHarold** [OvertureMaps/data#176] [Discussion]
### Category Feedback

 - Expected behaviour you didn't see 
 Hi, the change log mentions a lanes column in segments but I'm not seeing it, am I missing something?

**Comment by brad-richardson** [comment]
@BruceHarold Could you point me to where you saw that? We don't currently produce lanes data although it does exist in the schema.

**Reply by BruceHarold** [reply]
Hi, lanes is mentioned here:

https://docs.overturemaps.org/release/latest/#breaking-changes

**Comment by brad-richardson** [comment]
Lanes has been dropped from the schema until we can actually ship the populating data, see the latest 2024-11-13 release notes for more details
